backend/data/src/main/kotlin/io/tolgee/service/dataImport/StoredDataImporter.kt

```diff
@@ -154,6 +153,17 @@ class StoredDataImporter(
         keyService.deleteMultiple(otherKeys.map { it.value.id })
       }
     }
+    if (importSettings.onlyUpdateWithoutAdd) {
```

Comment:
Unfortunately, this whole feature will be much more complicated. In the UI, if you update the settings, it doesn't require re-uploading of the file. Also the numbers shown in the UI reflect the number or actually imported keys, so keys which are ignored by this operations should not be counted.

Because of this, you should 
1. Store if each import key should be imported or not when data are added to import
2. Show correct counts to user and remove these keys/translations from the data view.
3. Refresh this when import is applied (the existing data can change).
4. Accordingly ignore the translations and other data of ignored keys.